packages/grid/x-data-grid/src/hooks/features/editing/useGridCellEditing.ts

```diff
@@ -522,7 +515,8 @@ export const useGridCellEditing = (
     }
   }, [cellModesModelProp, updateCellModesModel]);
 
-  React.useEffect(() => {
+  // Run this effect synchronously so that the keyboard event can impact the yet-to-be-rendered input.
+  useEnhancedEffect(() => {
```

Comment:
Changing `useEffect` to `useLayoutEffect` here makes Chromium crash in some cases (at least the version that Playwright is using - 119.0.6045.9):

https://github.com/mui/mui-x/assets/13808724/bcf64467-0897-4425-bb03-f0541c7c12a1

I cannot reproduce it in Google Chrome 119.0.6045.105 though.

Reply:
The issue is gone after Playwright upgrade